src/Creative/InLine/Linear.php

```diff
@@ -106,7 +109,23 @@ public function skipAfter($time)
             $time = $this->secondsToString($time);
         }
 
-        $this->getDomElement()->firstChild->setAttribute('skipoffset', $time);
+        $this->getDomElement()->getElementsByTagName('Linear')->item(0)->setAttribute('skipoffset', $time);
+
+        return $this;
+    }
+
+
+    /**
+     * @param int|string $idRegistry
```

Comment:
Add vrsion of vast from where this element is available and section of specification to read about it.

Reply:
Done